src/main/java/zeenea/connector/property/PropertyDefinition.java

```diff
@@ -87,18 +99,19 @@ public static StringPropertyDefinition string(String code) {
    * @param label the label of the property definition
    * @return a new StringPropertyDefinition instance
    */
+  @Deprecated(since = "2.3.0", forRemoval = true)
```

Comment:
Et là également : 
```suggestion
  
  @Deprecated(
      since =
          "Deprecated since version 2.3.0, use string(name) instead. Scheduled for removal in version 3.0.0.",
      forRemoval = true)
```